lib/server/routes/buckets.js

```diff
@@ -1204,6 +1206,19 @@ BucketsRouter.prototype.removeFile = function (req, res, next) {
                 if (err) {
                   log.error('Error deleting pointer %s, reason: %s', pointer._id, err.message);
                 }
+                else {
+                  this.networkQueue.enqueueMessage({
+                    type: DELETING_FILE_MESSAGE,
+                    payload: { hash: pointer.hash }
+                  }, (err) => {
+                    if (err) {
+                      log.error('Error adding deleting task to the queue of shard: %s', pointer.shard);
+
+                      nextPointer();
```

Comment:
A callback should be only called once. `nextPointer` is being called twice (L1217, L1222).

As we were looking for a fire-and-forget behavior, consider deleting `nextPointer` call from here, letting the original `nextPointer` call unchanged

Reply:
Yep, completely agree.

Fixed.